rest-api-spec/src/main/resources/rest-api-spec/api/indices.flush_synced.json

```diff
@@ -2,7 +2,11 @@
   "indices.flush_synced":{
     "documentation":{
       "url":"https://www.elastic.co/guide/en/elasticsearch/reference/master/indices-synced-flush-api.html",
-      "description":"Performs a synced flush operation on one or more indices. Synced flush is deprecated and will be removed in 8.0. Use flush instead"
+      "description":"Performs a synced flush operation on one or more indices."
+    },
+    "deprecated" : {
+      "version" : "7.6.0",
+      "description" : "Synced flush is deprecated and will be removed in 8.0. Use flush instead"
```

Comment:
Missing period for the second sentence? :)

Reply:
@russcam In other places (like the [update_transform JSON spec][0]), the `deprecated` object was added at the `paths` objects.

Github won't let me comment there, but the `url` object would look something like this:

```
    "url":{
      "paths":[
        {
          "path":"/_flush/synced",
          "methods":[
            "POST",
            "GET"
          ],
          "deprecated":{
            "version":"7.6.0",
            "description":"[/_flush/synced] is deprecated and will be removed in 8.0. Use [/_flush] instead."
          }
        },
        {
          "path":"/{index}/_flush/synced",
          "methods":[
            "POST",
            "GET"
          ],
          "parts":{
            "index":{
              "type":"list",
              "description":"A comma-separated list of index names; use `_all` or empty string for all indices"
            },
            "deprecated":{
              "version":"7.6.0",
              "description":"[/{index}/_flush/synced] is deprecated and will be removed in 8.0. Use [/{index}/_flush] instead."
            }
          }
        }
      ]
    },
```

[0]: https://github.com/elastic/elasticsearch/blob/master/x-pack/plugin/src/test/resources/rest-api-spec/api/data_frame_transform_deprecated.update_transform.json

Reply:
Thanks @jrodewig. I think the deprecation should be applied at the top level for this and for transforms, otherwise the deprecation object needs to be repeated for each path, which seems superfluous. Based on https://github.com/elastic/elasticsearch/pull/51906#issuecomment-582274535 though, it looks like deprecated isn't implemented at this level currently.